fastapi_template/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/web/lifetime.py

```diff
@@ -120,6 +120,24 @@ def _setup_db(app: FastAPI) -> None:  # pragma: no cover
     app.state.db_session_factory = session_factory
 {%- endif %}
 
+{%- if cookiecutter.orm == "beanie" %}
+import beanie
+from motor.motor_asyncio import AsyncIOMotorClient
+from {{cookiecutter.project_name}}.db.models.dummy_model import DummyModel
+async def _setup_db(app: FastAPI) -> None:
+    client = AsyncIOMotorClient(
+        f"mongodb://{settings.db_user}:{settings.db_pass}"
```

Comment:
Please use yarl to build URLs.

Take a look at `db_url` property in settings.

Reply:
ok! thanks for the tips